src/services/projection/projections/needed.ts

```diff
@@ -130,7 +130,14 @@ export const calculateNeededForSkillForPeriod = (
           }
 
           // If there is no assignment for this day, the needed value is the role's start confidence
```

Comment:
I would put this comment under the else or the other way around, to keep the two comments separated